README.md

```diff
@@ -2,6 +2,7 @@ python-for-android
 ==================
 
 [![Build Status](https://travis-ci.org/kivy/python-for-android.svg?branch=develop)](https://travis-ci.org/kivy/python-for-android)
+[![Unit tests & build apps](https://github.com/kivy/python-for-android/workflows/Unit%20tests%20&%20build%20apps/badge.svg?branch=develop)](https://github.com/kivy/python-for-android/actions?query=workflow%3A%22Unit+tests+%26+build+apps%22)
```

Comment:
Anxious to see how it will look like this badge :+1:

Reply:
ahah, you could actually see it from my fork/branch https://github.com/AndreMiras/python-for-android/tree/feature/python2_drop_follow_up
But you will see soon enough as I'm merging, thanks for the review :)